.github/changeset-version.js

```diff
@@ -0,0 +1,4 @@
+const { execSync } = require("node:child_process");
```

Comment:
could you add a comment here why we do this? It looks odd without the added context.